charts/camunda-platform/test/unit/optimize/golden/deployment.golden.yaml

```diff
@@ -98,4 +98,17 @@ spec:
           failureThreshold: 5
           timeoutSeconds: 1
         volumeMounts:
-      volumes:
\ No newline at end of file
+      volumes:
+      initContainers:
+      - name: migration
+        image: "camunda/optimize:3.10.5"
+        command: ['./upgrade/upgrade.sh', '--skip-warning']
+        env:
+        - name: CAMUNDA_OPTIMIZE_ZEEBE_ENABLED
+          value: "true"
+        - name: CAMUNDA_OPTIMIZE_ZEEBE_PARTITION_COUNT
+          value: "3"
+        - name: OPTIMIZE_ELASTICSEARCH_HOST
+          value: "elasticsearch-master"
```

Comment:
This is not working with external Elasticsearch clusters anymore. If it's not hosted in the Kubernetes cluster directly

Reply:
Just for records, follow-up issue: https://github.com/camunda/camunda-platform-helm/issues/953